DropInDemo/Features/Drop In/DemoDropInViewController.swift

```diff
@@ -48,7 +48,7 @@ class DemoDropInViewController: DemoBaseViewController {
         dropInRequest.venmoDisabled = DemoSettings.venmoDisabled
         dropInRequest.cardDisabled = ProcessInfo.processInfo.arguments.contains("-CardDisabled")
         dropInRequest.shouldMaskSecurityCode = DemoSettings.maskSecurityCode
-        dropInRequest.cardholderNameSetting = DemoSettings.cardholderNameSetting
+        dropInRequest.cardholderNameSetting = BTFormFieldSetting.required
```

Comment:
Is this change supposed to be part of this PR?

Reply:
NOPE

Reply:
😂